.github/workflows/first-time-contributor.yml

```diff
@@ -0,0 +1,42 @@
+name: First time contributor
+
+on:
+  pull_request:
+    types: [opened]
+  issues:
+    types: [opened]
+
+jobs:
+  greet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Greet User 😃
+        uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Thanks for creating your first issue on OpenTelemetry JS!.
+            Someone should comment on your issue shortly.
+
+            If this issue is urgent, please mention `@opentelemetry/javascript-approvers` or `@opentelemetry/javascript-maintainers`.
+          pr-message: |
+            # First Time Contributor
+
+            Congratulations on your first contribution to OpenTelemetry JS!.
+            Please make sure you have read and completed the contribution checklist, which should have been used as the template when you created the PR.
+            If you ever need to refer to the checklist later, it is available at [.github/PULL_REQUEST_TEMPLATE.md](https://github.com/open-telemetry/opentelemetry-js/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+
+
+            ## Sign the Contributors Licence Agreement
+
+            All CNCF projects require contributors to sign a CLA.
+            You can sign as a company or an individual.
+            All pull requests run several checks, including checking if the user who created the pull request signed the CLA.
+            If you have not signed the CLA, the failed check will have a link you can follow in order to sign.
+            Once you have signed, come back to this PR and comment "I signed it,"" and the check will be rerun.
```

Comment:
"I signed it,""
there is double " sign and comma, is this on purpose ?

Reply:
Comma should be there as it is part of the parent sentence. Duplicate double quotes is a mistake. I'll update it to make it more clear that the content of the message is actually not important.